packages/common/src/discussions/_internal/DiscussionSchema.ts

```diff
@@ -0,0 +1,50 @@
+import { IConfigurationSchema } from "../../core";
+import {
+  ENTITY_ACCESS_SCHEMA,
+  ENTITY_CATEGORIES_SCHEMA,
+  ENTITY_NAME_SCHEMA,
+  ENTITY_TAGS_SCHEMA,
+} from "../../core/schemas/shared";
+
+export type DiscussionEditorType = (typeof DiscussionEditorTypes)[number];
+export const DiscussionEditorTypes = ["hub:discussion:edit"] as const;
+
+/**
+ * defines the JSON schema for a Discussion's editable fields
+ */
+export const DiscussionSchema: IConfigurationSchema = {
+  required: ["name"],
+  type: "object",
+  properties: {
+    name: ENTITY_NAME_SCHEMA,
+    summary: {
+      type: "string",
+    },
+    description: {
+      type: "string",
+    },
+    access: ENTITY_ACCESS_SCHEMA,
+    groups: {
```

Comment:
Looking at the specs in the issue, it doesn't look like you're exposing group editing anywhere in the "Details" panel. I assume this _might_ become relevant for the creation workflow (similar to projects), but you could remove it from the schema for now.